app/templates/suppliers/registered_address.html

```diff
@@ -34,12 +34,7 @@
 
   <div class="grid-row">
     <div class="column-one-whole">
-      {% with
-        heading = "What is your registered office address?",
-        smaller = true
-      %}
-        {% include 'toolkit/page-heading.html' %}
-      {% endwith %}
+      <h1 class="govuk-heading-l">$1</h1>
```

Comment:
Same again, sorry for the repetition - should this be `"What is your registered office address?"`

Reply:
```suggestion
      <h1 class="govuk-heading-l">What is your registered office address?</h1>
```